eventing/samples/k8s-events/README.md

```diff
@@ -0,0 +1,267 @@
+# Flow Example
+
+A simple function which receives and logs Kubernetes Events. This examples uses
+all the different components of the Knative Eventing stack with a `stub`
+ClusterBus and the `k8sevents` EventSource. The function is deployed using a
+Knative serving service, so it scales automatically as event traffic increases.
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+- [Docker](https://www.docker.com/) installed and running on your local machine,
+  and a Docker Hub account configured (you'll use it for a container registry).
+- The core Knative eventing tools installed. You can install them with:
+  ```shell
+  kubectl apply -f https://storage.googleapis.com/knative-releases/latest/release-eventing.yaml
```

Comment:
This url is now `https://storage.googleapis.com/knative-releases/eventing/latest/release.yaml`